packages/cli/src/commands/alias/ls.ts

```diff
@@ -22,13 +23,19 @@ export default async function ls(
 ) {
   const { output } = client;
   const { '--next': nextTimestamp } = opts;
+  const { '--limit': limit } = opts;
```

Comment:
I know you're copying `--next`, but passing `--next` and `--limit` around seems wrong. It would be cool if those were resolved into `nextTimestamp` and `limit`, then passed into `ls()`.

Reply:
Agreed would be good to refactor this and reduce the amount of copied code, also would reduce the amount of boilerplate in the tests. I'm 50/50 if I want to do this in this PR or just ship this and add that to the backlog at this point.

Reply:
After discussing with Andy, I didn't realize that `opts` is actually `argv`, thus the existing code is good to go.